### PR TITLE
Update project_binary_search_trees.md

### PR DESCRIPTION
**Pull request by jakemadash**
I think what's being asked for in these method tasks is a little confusing. See the below excerpt from the discord. Now that I've seen this, I understand, but I can definitely understand Melvin's confusion. Maybe an example or just a mention of the nodes being modified via the function in some way would help clarify? It's a bit hard to parse as is.

melvin — 09/26/2022
Binary Search Trees Task 7 Write inorder, preorder, and postorder functions that accept a function parameter. Each of these functions should traverse the tree in their respective depth-first order and yield each node to the provided function given as an argument. The functions should return an array of values if no function is given. What does  The functions should return an array of values if no function is given. mean? The functions should take an input and if no input is provided they should return an array of values? Where do the values come from?

nevz — 09/26/2022
The values are each node you've visited during the traversal, so each time you've visited a node, the value of that node will be pushed to an array. 

If you've passed a function to those methods for example levelorder((el) => el * 2)) then each value will be multiplied by 2. Otherwise, you will just return the array without any modification.

You should be returning something like:
[21, 10, 156, 4] // levelorder IF no function
[42, 20, 312, 8] // levelorder IF there is a function (el) => el * 2

Complete the following REQUIRED checkboxes:

-   [ ] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [ ] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [ ] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [ ] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

**2. This PR:**

**3. Additional Information:**

### CODE REVIEW

**Comment by nevz9** [comment]
I also misunderstood this and what I've said was what I implemented, but looking at the instructions again, it seems like it only wants you to return an array **if** a function was not given. Otherwise, you would just call the function passing it the current visited node.

I'm not sure if I can give an example that won't give out the answer, but hopefully someone can clarify.

**Comment by zachmmeyer** [comment]
I feel like we have a similar issue on the ruby side. The instructions there are to use a block, yet looking at a lot of solutions I don't believe a block is being used at all. It certainly isn't in mine.

**Comment by thatblindgeye** [comment]
@jakemadash Was this PR meant to implement a fix? I'm not sure what "See description" in the PR changes means.

**Comment by KevinMulhern** [comment]
Closing due to no response
